homeassistant/components/insteon.py

```diff
@@ -35,10 +34,23 @@ def setup(hass, config):
         return False
 
     import insteon
```

Comment:
It would be a bit cleaner to `import insteon` after validating the config. That way, it is not imported needlessly if there is something wrong with the config.